roles/nut/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: "Install Network UPS tool"
+  ansible.builtin.apt:
+    update_cache: yes
+    name: nut
+    state: present
+
+- name: "Retrieve vault credentials for NUT"
```

Comment:
This task should probably have `become: false` because we don't need to run as root to make an API call to Vault ^^ (and probably won't have the rights to do it)

Reply:
Resolved with [fefab0a](https://github.com/rezoleo/ansible-playbooks/pull/6/commits/fefab0ae88a104176dabcfbb748fd0f79651b91a)